src/backend/backend.ts

```diff
@@ -50,12 +43,16 @@ window.addEventListener('message', function(event) {
       const highlightStr = '[batarangle-id=\"' +
         event.data.message.message.node.id + '\"]';
 
-      (<HTMLElement>document.querySelector(highlightStr)).scrollIntoView();
+      const element: any = <HTMLElement>document.querySelector(highlightStr);
```

Comment:
Do we need a cast to HTML element here? does querySelector return any or something else?